app/services/submit_to_app_store.rb

```diff
@@ -31,7 +31,12 @@ def submit(submission)
     # able to support re-sending/submitting an appplication so we can ignore
     # put requests
     post_manager = AppStoreClient.new
-    post_manager.post(payload)
+
+    if submission.is_a?(PriorAuthorityApplication)
+      submission.app_store_updated_at.present? ? post_manager.put(payload) : post_manager.post(payload)
```

Comment:
Given it's not `post`ing, maybe we stop calling this variable `post_manager`. Maybe just `client`?

Reply:
yeah agreed